perception/radar_crossing_objects_noise_filter/test/test_radar_crossing_objects_noise_filter/test_radar_crossing_objects_filter_is_noise.cpp

```diff
@@ -0,0 +1,208 @@
+// Copyright 2023 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "radar_crossing_objects_noise_filter/radar_crossing_objects_noise_filter_node.hpp"
+
+#include <gtest/gtest.h>
+
+std::shared_ptr<radar_crossing_objects_noise_filter::RadarCrossingObjectsNoiseFilterNode> get_node(
+  float angle_threshold, float velocity_threshold)
+{
+  rclcpp::NodeOptions node_options;
+
+  node_options.parameter_overrides(
+    {{"angle_threshold", angle_threshold}, {"velocity_threshold", velocity_threshold}});
+  auto node =
+    std::make_shared<radar_crossing_objects_noise_filter::RadarCrossingObjectsNoiseFilterNode>(
+      node_options);
+  return node;
+}
+
+std::shared_ptr<autoware_auto_perception_msgs::msg::DetectedObject> get_object(
+  float twist_x, float twist_y, float twist_z, float orientation_x, float orientation_y,
+  float orientation_z, float orientation_w, float pose_x, float pose_y)
+{
+  auto ret = std::make_shared<autoware_auto_perception_msgs::msg::DetectedObject>();
```

Comment:
[MUST]
Could you change the variable from float to double?
Is there any reason to use float.

[MUST] ret -> object
Please avoid variable names that don't make sense.

[MUST]
In addition to it, if you can, please suppress the issue of "New issue: Excess Number of Function Arguments".
I recommend to use `tier4_autoware_utils`

https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp#L291
https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp#L554
https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_autoware_utils/src/geometry/geometry.cpp#L60

In summary, example is here

```cpp
autoware_auto_perception_msgs::msg::DetectedObject get_object(
  Vector3 velocity, geometry_msgs::msg::Point position, geometry_msgs::msg::Quaternion orientation){
    autoware_auto_perception_msgs::msg::DetectedObject object;
    object.kinematics.twist_with_covariance.twist.linear = velocity;
    object.kinematics.pose_with_covariance.pose.position = position;
    object.kinematics.pose_with_covariance.pose.orientation = orientation;
  }

// for use
auto velocity = tier4_autoware_utils::createVector3(40.0, 30.0, 0.0);
auto position = tier4_autoware_utils::createPoint(1.0, 0.0, 0.0);
auto orientation = tier4_autoware_utils::createQuaternion(1.0, 1.0, 1.0, 0.0);
auto object = get_object(velocity, position, orientation);

```